### PR TITLE
Add multilabel classification class

### PR DESCRIPTION
**Pull request by ddofer**
The main change is in the _compute_loss function, where I replaced F.cross_entropy with F.binary_cross_entropy_with_logits. This is suitable for multilabel classification tasks, where the model must predict multiple labels per input.

### CODE REVIEW

**Comment by agemagician** [comment]
@hazemessamm could you please take a look?

Do you think it is better to use BCEWithLogitsLoss or binary_cross_entropy_with_logits in this case ?

**Comment by hazemessamm** [comment]
> @hazemessamm could you please take a look?
> 
> Do you think it is better to use BCEWithLogitsLoss or binary_cross_entropy_with_logits in this case ?

It's better to use `binary_cross_entropy_with_logits` for two reasons, the first one is that the rest of the classes are using the `functional` module so it's consistent with them and the second reason is that we will avoid re-initializing `BCEWithLogitsLoss` instance everytime we call `_compute_loss`, if we want to avoid re-initializing it we will need to initialize it in the `__init__` but it will be inconsistent with the implemented classes.

**Comment by hazemessamm** [comment]
Hello @ddofer,

Thank you for your PR.

Can you please add the class in the `src/models/__init__.py` and in `src/__init__.py` and make sure that you formatted `convbert_multilabel_classification.py` using `black`?

**Comment by ddofer** [comment]
Done (added to the 2 INITs) and python black formatted

**Comment by hazemessamm** [comment]
Hello @ddofer, Could you please remove the `author` comment because your username will be already visible in the commit history and also for consistency with other files as we did not put who is the author in any file because as I mentioned it will be already visible in the commit history.

**Comment by ddofer** [comment]
Done @hazemessamm

**Review by hazemessamm — APPROVED** [review]
LGTM!

Thank you for your PR. :)
